src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/handler/PaymentMessageHandler.java

```diff
@@ -42,11 +42,20 @@ public void handlePaymentMessage(CreatePaymentMessage paymentMessage, String mes
         createPayment(paymentMessage, messageId);
 
         if (paymentMessage.isExceptionRecord) {
-            ccdClient.completeAwaitingDcnProcessing(
-                paymentMessage.ccdReference,
-                paymentMessage.service,
-                paymentMessage.jurisdiction
-            );
+            try {
+                ccdClient.completeAwaitingDcnProcessing(
+                    paymentMessage.ccdReference,
+                    paymentMessage.service,
+                    paymentMessage.jurisdiction
+                );
+            } catch (FeignException exception) {
+                log.debug(
+                    "Failed to call 'completeAwaitingDcnProcessing' method. CCD response: {}",
+                    exception.responseBody().map(b -> new String(b.array())).orElseGet(exception::getMessage)
```

Comment:
what's `b`?

Reply:
respones **B**ody. i gathered there's no need to be suuuper specific here